### ISSUE TITLE
Automatically update gradient when layer name is changed.

### ISSUE THREAD

**Issue by LPZilva** [larsenwork/sketch-easing-gradient#7]
**Is your feature request related to a problem? Please describe.**
Not directly related to a problem, but a shortcut to the current plugin flow.

**Describe the solution you'd like**
Whenever I rename a layer name, the plugin would fetch data changed, updating the gradient accordingly.

**Describe alternatives you've considered**
Nothing to add.

**Additional context**
Nothing to add.

**Comment by larsenwork** [comment]
Yeah, not too sure about this since it's maybe a bit too magic and in the current flow doing what you describe is still possible — only you have to press `cmd + alt + g` after changing the layer name to update the gradient. 

Did you have any examples of other plugins that does this kinda thing on layer name edit?

**Comment by LPZilva** [comment]
I see - [Paddy](https://github.com/DWilliames/paddy-sketch-plugin) does what I'm talking about, and it's pretty neat! Have a look.

https://github.com/DWilliames/paddy-sketch-plugin
